src/main/java/com/unichristus/libraryapi/model/User.java

```diff
@@ -0,0 +1,26 @@
+package com.unichristus.libraryapi.model;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Table(name = "users")
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
```

Comment:
```suggestion
    @GeneratedValue(strategy = GenerationType.UUID)
```
aqui deveria ser UUID

Reply:
Pode utilizar a classe do meu PR como exemplo se quiser